src/Appwrite/Auth/OAuth2/Slack.php

```diff
@@ -56,16 +56,15 @@ public function getLoginURL(): string
     protected function getTokens(string $code): array
     {
         if (empty($this->tokens)) {
-            // https://api.slack.com/docs/oauth#step_3_-_exchanging_a_verification_code_for_an_access_token
             $this->tokens = \json_decode($this->request(
                 'GET',
-                'https://slack.com/api/oauth.access?' . \http_build_query([
+                'https://slack.com/api/oauth.v2.access?' . \http_build_query([
                     'client_id' => $this->appID,
                     'client_secret' => $this->appSecret,
                     'code' => $code,
                     'redirect_uri' => $this->callback
                 ])
-            ), true);
+            ), true)['authed_user'];
```

Comment:
Accessing like this is dangerous and can throw an exception leading to a 500 error. Let's be a bit more defensive and use `??`